integreat_cms/api_client_firebase/push_notification_sender.py

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
Could you delete the old file [integreat_cms/cms/utils/push_notification_sender.py](https://github.com/digitalfabrik/integreat-cms/blob/enhancement/push_notification_sender_api/integreat_cms/cms/utils/push_notification_sender.py)? Otherwise we will just have the same code twice.